pkg/resource/processing_job/testdata/test_suite.yaml

```diff
@@ -1,17 +1,174 @@
   - name: "Processing job create tests"
     description: "Part of processing job CRD tests."
     scenarios:
-     - name: "Create=InvalidInput"
-       description: "Given one of the parameters is invalid, ko.Status shows a terminal condition"
-       given:
-         desired_state: "processing_job/v1alpha1/pj_invalid_before_create.yaml"
-         svc_api:
-           - operation: CreateProcessingJobWithContext
-             error:
-               code: InvalidParameterValue
-               message: "The processing job name must not include a special character."
-       invoke: Create
-       expect:
-         latest_state: "processing_job/v1alpha1/pj_invalid_create_attempted.yaml"
+      - name: "Create=InvalidInput"
+        description: "Given one of the parameters is invalid, ko.Status shows a terminal condition"
+        given:
+          desired_state: "v1alpha1/create/desired/invalid_before_create.yaml"
+          svc_api:
+            - operation: CreateProcessingJobWithContext
+              error:
+                code: InvalidParameterValue
+                message: "The processing job name must not include a special character."
+        invoke: Create
+        expect:
+         latest_state: "v1alpha1/create/observed/invalid_create_attempted.yaml"
          error: resource is in terminal condition
+      - name: "Create=Valid"
+        description: "Create a new processing job successfully (ARN in status)."
+        given:
+          desired_state: "v1alpha1/create/desired/success_before_create.yaml"
+          svc_api:
+            - operation: CreateProcessingJobWithContext
+              output_fixture: "sdkapi/create/success_create.json"
+        invoke: Create
+        expect:
+          latest_state: "v1alpha1/create/observed/success_after_create.yaml"
+          error: nil
+      - name: "Create=Valid2"
+        description: "Create a new processing job successfully (ARN in status)."
+        given:
+          desired_state: "v1alpha1/create/desired/success_before_create_variation.yaml"
+          svc_api:
+            - operation: CreateProcessingJobWithContext
+              output_fixture: "sdkapi/create/success_create.json"
+        invoke: Create
+        expect:
+          latest_state: "v1alpha1/create/observed/success_after_create_variation.yaml"
+          error: "nil"
+  - name: "Processing job readOne tests"
+    description: "Testing the readOne operation"
+    scenarios:
+      - name: "ReadOne=MissingRequiredField"
+        description: "Testing readOne when required field is missing. No API call is made and returns error."
+        given: 
+          desired_state: "v1alpha1/readone/desired/missing_required_field.yaml"
+        invoke: ReadOne
+        expect:
+          error: "resource not found"
+      - name: "ReadOne=NotFound"
+        description: "Testing readOne when Describe fails to find the resource on SageMaker"
+        given: 
+          desired_state: "v1alpha1/readone/desired/after_create.yaml"
```

Comment:
nit: after_create.yaml is not a clear enough filename IMO, not sure what it means

Reply:
changed file name